components/coding/profileV2/AssignmentSectionComponent.tsx

```diff
@@ -56,12 +81,18 @@ export default function AssignmentsSection({}: AssignmentSectionComponentProps)
           userAssignments.map((assignment, index) => {
             return (
               <div
-                className={`grid grid-cols-5 my-2 text-sm text-center md:grid-cols-12 md:text-lg place-items-center ${assignment}`}
+                className={`grid grid-cols-7 my-2 text-sm text-left md:grid-cols-12 md:text-lg md:place-items-center  ${assignment}`}
               >
-                <p className="col-span-1">{index + 1}.</p>
-                <p className="col-span-1 md:col-span-4">
-                  {assignment.coding_assignment.assignment_name}
-                </p>
+                <p className="place-item-left col-span-1">{index + 1}.</p>
```

Comment:
what does place-item-left do? i'm not 100% sure it exists in tailwind, tried looking for it in the documentation.. lmk if you disagree

https://tailwindcss.com/docs/place-items#stretch

Reply:
it was a mistake -- was supposed to be place-items-start -- but now that I removed it, the change is so negligible that I don't think it's worth it and took it out!